lib/mpl_toolkits/axisartist/tests/test_floating_axes.py

```diff
@@ -23,6 +23,7 @@ def test_subplot():
 # remove when image is regenerated.
 @image_comparison(['curvelinear3.png'], style='default', tol=5)
 def test_curvelinear3():
+    plt.rcParams.update({"xtick.direction": "in", "ytick.direction": "in"})
```

Comment:
missing or intentional?
```suggestion
    # Remove this line when this test image is regenerated.
    plt.rcParams.update({"xtick.direction": "in", "ytick.direction": "in"})
```